src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/form.html.twig

```diff
@@ -1,11 +1,15 @@
+{% if form is not defined %}
+    {% set form = hookable_metadata.context.form %}
+{% endif %}
+
 {% form_theme form '@SyliusAdmin/shared/form_theme.html.twig' %}
 
-<div class="container-xl" {{ attributes }}>
+<div class="container-xl" {% if attributes is defined %} {{ attributes }} {% endif %}>
```

Comment:
```suggestion
<div class="container-xl" {{ attributes|default('') }}>
```
I did not remember if `default` mute the undefined var but if it does this is a shorter way to do it.

Reply:
yes, default mute the undefined.